doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-% Author: Pavol Loffay
-% Project: Master thesis - Hawkular alert prediction
-% Date 20.2.2015
-
-% Cross-Referencing Conventions:
-% chap: chapter
-% sec: section
-%
-% appen: everything from appendix, e.g. appen:img:foo
-%
-% eq: equation
-% img: image
-% tab: table
-% alg: algorithms
-% item: item in itemize
-%
-% example \label{img:foo-foo}
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Introduction} \label{chap:introduction}
-It is important to assure an application health and reliability when driving successful business on the internet. One
-can achieve that by monitoring subjected resources and setting up clever alerting system.
-
-These features are offered in many monitoring systems, however being predictive in this area can even prevent
-undesirable states and most importantly gives administrators more time for reacting to such events. For instance it
-can decrease downtime of an application or ability to load balance workload in advance by horizontal scaling of
-targeted services.
-
-Alerting system are sophisticated and can be composed by many conditions. This work focuses on predicting future
```

Comment:
Je to myslene tak ze alertovaci system sa moze skladat z vela podmienok. (podmienky sa vyhodnocuju a podla toho sa vyvola alert)